examples/http.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 Google Inc.
```

Comment:
Since this file is basically main.go from the root of the repo, maybe it should still be 2017.